defs.h

```diff
@@ -4,30 +4,9 @@
  * Global definitions and Macros.
  */
 
-#define DMAXX					40
-#define DMAXY					40
-
-#define LIGHTSIZE				6912 // 27 * 256
-#define NO_LIGHT				-1
-
-#define GMENU_SLIDER			0x40000000
-#define GMENU_ENABLED			0x80000000
-
-// must be unsigned to generate unsigned comparisons with pnum
-#define MAX_PLRS				4
-
-#define MAX_CHARACTERS			99
-#define MAX_LVLS				24
-#define MAX_LVLMTYPES			24
 #define MAX_SPELLS				52
 #define MAX_SPELL_LEVEL			15

```

Comment:
Why didn't we move some of these?

Reply:
more complicated to move, save them for round 2